index.bs

```diff
@@ -1431,6 +1431,12 @@ Today writing to {{Document/cookie|document.cookie}} in contexts where script-in
 
 Likewise, today reading {{Document/cookie|document.cookie}} in contexts where script-initiated cookie-reading is disallowed typically returns an empty string. However, a cooperating web server can verify that server-initiated cookie-writing and cookie-reading work and report this to the script (which still sees empty string) and the script can use this information to infer that script-initiated cookie-reading is disallowed.
 
+<!-- ============================================================ -->
+## Clear cookies ## {#clear-cookies}
+<!-- ============================================================ -->
+
```

Comment:
This should be called out as "non-normative" (e.g. just add `*This section is non-normative.*` here)

Reply:
Thanks for pointing this out!